### ISSUE TITLE
Task table issue

### ISSUE THREAD

**Issue by ikrc-teamcity** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#670]
Just wanted to see if this is a known issue / limitation for any reason.

We have a table [dbo].[Task], table script below for reference.

If this table is not excluded the following errors appear:

'FakeDbSet<TEntity>.AddRangeAsync(params TEntity[])': return type must be 'Task' to match overridden member 'DbSet<TEntity>.AddRangeAsync(params TEntity[])'

**Table Script:**
```sql
CREATE TABLE Task
(
    TaskId  BIGINT NOT NULL
    CONSTRAINT PK_Task PRIMARY KEY CLUSTERED (TaskId ASC)
);
GO
```
Thanks!

**Comment by sjh37** [comment]
This was solved by adding 'Task' as a reserved keyword. This will be in the next release.
For now you can grab yourself the latest code from [EF.Reverse.POCO.v3.ttinclude](https://raw.githubusercontent.com/sjh37/EntityFramework-Reverse-POCO-Code-First-Generator/master/EntityFramework.Reverse.POCO.Generator/EF.Reverse.POCO.v3.ttinclude) and paste it into your project.

**Comment by sjh37** [comment]
This has now been released in v3.4.0
